src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,16 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        excavator.doWork();
+        excavator.stopWork();
```

Comment:
first you should send all machines to work and then stop them

Reply:
Done